src/dstack/_internal/core/models/fleets.py

```diff
@@ -100,6 +100,10 @@ class InstanceGroupParams(CoreModel):
         Optional[InstanceGroupPlacement],
         Field(description="The placement of instances: `any` or `cluster`"),
     ] = None
+    reservation: Annotated[
+        Optional[str],
+        Field(description="The existing reservation for the instances"),
+    ] = None
```

Comment:
Just pointing out another client backward incompatibility that we need to address.